components/export/ui/src/main/resources/PhenoTips/ExportFilter.xml

```diff
@@ -37,231 +37,34 @@
-    #else
-      #set ($query = $query.concat(' or '))
-    #end
-    #set ($query = $query.concat('owner.value = ?'))
-    #set ($discard = $parameters.add($services.model.resolveDocument($owner).toString()))
-  #end
-#end
-#if (!$isFirstOwner)
-  #set ($query = $query.concat(')'))
-  #set ($query = ", BaseObject ownerobj, StringProperty owner" + $query + " and ownerobj.className = '$ownerClassName' and owner.name = 'owner' and owner.id.id=ownerobj.id and ownerobj.name = doc.fullName")
-#end
-#set ($isFirstAuthor = true)
-#foreach ($author in $request.getParameterValues('doc.author'))
-  #if ($author != '')
-    #if ($isFirstAuthor)
-      #set ($query = $query.concat(' and ('))
-      #set ($isFirstAuthor = false)
-    #else
-      #set ($query = $query.concat(' or '))
-    #end
-    #if ($author.contains(':'))
-      #set ($query = $query.concat('doc.author = ? or '))
-      #set ($discard = $parameters.add($stringtool.substringAfter($author, ':')))
-    #end
-    #set ($query = $query.concat('doc.author = ?'))
-    #set ($discard = $parameters.add($author))
-  #end
-#end
-#if (!$isFirstAuthor)
-  #set ($query = $query.concat(')'))
-#end
-#set ($isFirstPhenotype = true)
-#set ($joinOperator = "$!{request.get('phenotype/join_mode').toUpperCase()}")
-#if ($joinOperator != 'AND' &amp;&amp; $joinOperator != 'OR')
-  #set ($joinOperator = 'AND')
-#end
-#foreach ($phenotype in $request.getParameterValues('phenotype'))
-  #if ($phenotype != '')
-    #if ($isFirstPhenotype)
-      #set ($isFirstPhenotype = false)
-      #if ("$!request.phenotype_subterms" == 'no')
-        #set ($query = ", DBStringListProperty phenotype ${query} and phenotype.id.id = obj.id and phenotype.id.name = 'phenotype' and (? in elements(phenotype.list)")
-      #else
-        #set ($query = ", DBStringListProperty phenotype ${query} and phenotype.id.id = obj.id and phenotype.id.name = 'extended_phenotype' and (? in elements(phenotype.list)")
-      #end
-    #else
-      #set ($query = $query.concat(" ${joinOperator} ? in elements(phenotype.list)"))
-    #end
-    #set ($discard = $parameters.add($phenotype))
-  #end
-#end
-#if (!$isFirstPhenotype)
-  #set ($query = $query.concat(')'))
-#end
-#set ($isFirstDisease = true)
-#set ($joinOperator = "$!{request.get('omim_id/join_mode').toUpperCase()}")
-#if ($joinOperator != 'AND' &amp;&amp; $joinOperator != 'OR')
-  #set ($joinOperator = 'AND')
-#end
-#foreach ($disease in $request.getParameterValues('omim_id'))
-  #if ($disease != '')
-    #if ($isFirstDisease)
-      #set ($query = ", DBStringListProperty disease ${query} and disease.id.id = obj.id and disease.id.name = 'omim_id' and (? in elements(disease.list)")
-      #set ($isFirstDisease = false)
-    #else
-      #set ($query = $query.concat(" ${joinOperator} ? in elements(disease.list)"))
-    #end
-    #set ($discard = $parameters.add($disease))
-  #end
+#set ($extra = " $!{request.filterWhere} ")
+#set ($params = [])
+#if ("$!request.space" != '')
```

Comment:
From nowhere... that's why the filtering didn't work properly in Push and Export for non-admins. Sasha reported the issue and it was fixed in 67b4373 .